ci/ci_cli.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+import subprocess
+
+# import sdk_install
+import logging
+
+from concurrent import futures
+
+MAX_WORKERS = 10
+
+JOB_TYPE_INSTALL = 'install'
+JOB_TYPE_DIAGNOSTIC = 'diagnostic'
+
+JOB_TYPES = [JOB_TYPE_INSTALL, JOB_TYPE_DIAGNOSTIC]
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+def install(service_name):
+    import sdk_install
+
+    log.info("Starting %s installation.", service_name)
+
+    sdk_install.install(
+        service_name,
+        service_name,
+        0,
+        package_version=sdk_install.PackageVersion.LATEST_UNIVERSE
+    )
+
+
+def diagnostic(service_name):
+    log.info("Starting %s diagnostic.", service_name)
+
+    src = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        '..', 'python', 'create_service_diagnostics_bundle.sh'
+    )
+
+    cmd = '{src} --package-name={service} --service-name={service} --yes'.format(
+        src=src,
+        service=service_name)
+
+    process = subprocess.Popen(
+        [cmd],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        shell=True,
+        universal_newlines=True)
+
+    # print execution output
+    while True:
+        output = process.stdout.readline()
+        if output == '' and process.poll() is not None:
+            break
+        if output:
+            print(output.strip())
+
+
+def get_args() -> (str, list):
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--type",
+        choices=JOB_TYPES,
+        required=True,
+        help='Job type'
+    )
+
+    parser.add_argument(
+        'services',
+        nargs='+',
+        default='',
+        help='Services list'
+    )
+    args = parser.parse_args()
+
+    jobs = {
+        JOB_TYPE_INSTALL: install,
+        JOB_TYPE_DIAGNOSTIC: diagnostic,
+    }
+
+    funcs = [v for k, v in jobs.items() if k == args.type]
+
+    return funcs[0], args.services
```

Comment:
Why do you need `funcs`? Why not just `return jobs[args.type], args.services`?

Reply:
in a case, if args.types not in the jobs list (for instance add new option without function)

Reply:
changed